crnlib/crn_clusterizer.h

```diff
@@ -53,7 +53,7 @@ class clusterizer {
       root.m_total_weight += weight;
       root.m_vectors.push_back(i);
 
-      ttsum += v.dot(v) * weight;
+      ttsum += (double)v.dot(v) * (double)weight;

```

Comment:
Can we just disable float/double precision warnings please? It simply doesn't matter. This is a lossy image compression program.

Reply:
This is CodeQL static analysis report, not a compiler warning.

Reply:
I know that. It should be configurable.